playbooks/ORCHESTRATION.md

```diff
@@ -0,0 +1,219 @@
+| ----------- | ----------- | ----------- | 
+| GITHUB_USER      | The GitHub user, used to create PRs and commit messages       | asraa       |
+| BRANCH   | The working branch, in case of staging script or configuration changes.        | main        |
+| LOCAL   | If enabled, keeps git state dirty and does not create pull requests. Used to run root signing locally for testing.       |         |
+| REPO   | Specifies the repository folder, see [Configuration](#configuration)       |   `ceremony/2022-02-22`      |
+| PREV_REPO   | If set, this specifies a previous repository used to chain a following root signing event (copies previous hardware keys, etc).       |    `ceremony/2022-01-22`     |
+| SNAPSHOT_KEY   | The GCP KMS online key for snapshotting.    |     `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot`    |
+| TIMESTAMP_KEY   | The GCP KMS online key for timestamping.    |  `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/timestamp`  |
+| REKOR_KEY   | The GCP KMS online key for rekor delegation.       |     `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/rekor`    |
+| STAGING_KEY   | The GCP KMS online key for the staging delegation.       |   `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/staging`      |
+| REVOCATION_KEY   | The GCP KMS online key for the revocation delegation, containing any targets that were explicitly revoked.       |    `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/revocation`     |
+
+## Configuration
+
+Each root signing event occurs inside a folder `ceremony/YYYY-MM-DD/` named by the date the signing event started. This folder contains:
+* A `keys/` subfolder containing subdirectories named after Yubikey serial numbers, and containins public key PEMs, key certificates, and device certificates attesting to the hardware key. See [PIV attestation](https://developers.yubico.com/PIV/Introduction/PIV_attestation.html). 
+* A `repository/` subfolder containing the finalized TUF repository metadata.
+* A `staging/` subfolder present during root signing events with staged metadata before publishing.
+
+### Targets and Delegation configuration
+
+For top-level targets and delegation files, a YAML configuration file named `role-metadata.yml` specifies the target files for the role and their [custom metadata](https://theupdateframework.github.io/specification/latest/#custom). We use [Sigstore custom metadata](https://github.com/sigstore/sigstore/blob/ec8f2d403e07a392ea363560d21c31aaee57ba0f/pkg/tuf/client.go#L95) to allow Sigstore clients to identity usage of the targets. Target files are customarily located in the [targets/] subfolder. For example, this defines the target file located at `targets/fulcio_v1.crt.pem` with custom metadata to indicate Fulcio root usage.
+
+```yaml
+targets/fulcio_v1.crt.pem:
+  sigstore:
+    usage: Fulcio
+    status: Active
+```
```

Comment:
Not for docs: Another thing that might be useful to verify with a verification script, that usage and status are set for all targets

Reply:
It's not necessary for usage and status to be set, however. It's optional.

Filed https://github.com/sigstore/root-signing/issues/294 to track